code/web/src/modules/styleSurvey/surveyChoices.js

```diff
@@ -26,7 +24,10 @@ const SurveyChoices = ({ choices, handleChoice, choiceCategory }) => {
       margin: '5px',
       padding: '3px'
     }
-    return selectionStatus ? selectedStyle : unselectedStyle
+
+    // const choice = document.getElementById(`${choiceId}`)
+
+    return false ? selectedStyle : unselectedStyle
   }
```

Comment:
Could quite get styles to toggle for the selected images, hopefully is a small enough bug to fix